README.md

```diff
@@ -148,6 +150,7 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 
 | Company | Description | Locations |
 | :------ | :---------- | :-------- |
+| [Adamant Sec](https://adamantsec.com/) | Pentesting, consulting and audit. | `Porto` |
```

Comment:
Hi @speeddragon, is the company named **Adamant Sec** or just **Adamant**?

Reply:
Registered name is **Adamant Sec**.